virttest/utils_ids.py

```diff
@@ -61,7 +61,14 @@ def _get_subid(id_filepath, name, info):
     if not entry:
         raise TestError("No entry for %s found in %s" % (name, id_filepath))
 
-    entry = entry[0].split(":")
+    matched_entry = None
```

Comment:
Thank you @chunfuwen I wonder if we could have a shorter solution like the following:

```
entry = [l for l in result.stdout_text.split("\n") if name in l and name == l.split(":")[0]]
```

(I think `name in l` is still necessary to avoid an issue if there were empty or malformed lines.)

Reply:
updated